drake/systems/plants/collision/BulletModel.cpp

```diff
@@ -299,6 +299,40 @@ namespace DrakeCollision
     return c.getResults();
   }
 
+  bool BulletModel::collidingPointsCheckOnly(const vector<Vector3d>& points,
+                                             double collision_threshold)
+  {
+    // Create sphere geometry
+    btSphereShape bt_shape(collision_threshold);
+
+    // Create Bullet collision object
+    btCollisionObject bt_obj;
+    btCollisionObject* bt_obj_ptr = static_cast<btCollisionObject*>(&bt_obj);
```

Comment:
Strange - why would I have done that? Anyway, thanks for fixing it!

On Tue, Dec 1, 2015 at 9:09 AM MarcoCar <notifications@github.com> wrote:

> In drake/systems/plants/collision/BulletModel.cpp
> <https://github.com/RobotLocomotion/drake/pull/1467#discussion_r46281364>:
>
> > @@ -299,6 +299,40 @@ namespace DrakeCollision
> >      return c.getResults();
> >    }
> >
> > +  bool BulletModel::collidingPointsCheckOnly(const vector<Vector3d>& points,
> > +                                             double collision_threshold)
> > +  {
> > +    // Create sphere geometry
> > +    btSphereShape bt_shape(collision_threshold);
> > +
> > +    // Create Bullet collision object
> > +    btCollisionObject bt_obj;
> > +    btCollisionObject* bt_obj_ptr = static_cast<btCollisionObject*>(&bt_obj);
>
> @avalenzu <https://github.com/avalenzu> yes, it doesn't seem to be
> necessary so I removed it. Thanks for reviewing the code.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RobotLocomotion/drake/pull/1467/files#r46281364>.
>